README.md

```diff
@@ -0,0 +1,15 @@
+# Robocode
```

Comment:
Just a nitpick: I don't think it belongs here. Appdata already points to project's website but I'm not going to stop you!

Reply:
I don't understand, this is just a simple readme for the flatpak project. I don't think we should remove it.